ops/compile_test_runner/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+
+[package]
+name = "deno_ops_compile_test_runner"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+readme = "README.md"
+repository.workspace = true
+description = "Proc macro for writing Deno Ops"
```

Comment:
Nitpick: looks like the description wasn't updated. Also can we somehow mark this crate so it doesn't get published by accident?